src/ocm/components/clusters/DeleteClusterModal.tsx

```diff
@@ -5,16 +5,31 @@ type DeleteClusterModalProps = {
   name: string;
   onClose: () => void;
   onDelete: () => void;
+  isOpen: boolean;
+  isDeleteInProgress?: boolean;
 };
 
-const DeleteClusterModal: React.FC<DeleteClusterModalProps> = ({ name, onClose, onDelete }) => (
+const DeleteClusterModal: React.FC<DeleteClusterModalProps> = ({
```

Comment:
Can we remove the 'X' button disappear?
The modal shouldn't be dismissable when the operation is in progress.

Reply:
We can. We can also disable the "Cancel" button...